documentation/docs/02-layouts.md

```diff
@@ -113,64 +113,60 @@ src/routes/
 └ __layout-foo.svelte
 ```
 
-#### Resetting layouts
+#### Inheritance chains
 
-Any page, at any depth, can reset to the home layout, ignoring any intermediate `__layout` components by referencing the special `~` layout:
+Layouts can themselves choose which other layouts they inherit from. Ordinarily, they will inherit from any default layouts 'above' them in the tree, but they can inherit named layouts using the same logic as pages. For example, `a/b/c/page.svelte` will inherit from `__layout-foo.svelte` and `a/b/c/__layout@foo.svelte` but _not_ `__layout.svelte` or `a/b/__layout.svelte`.
 
 ```
 src/routes/
 ├ a/
 │ ├ b/
 │ │ ├ c/
-│ │ │ └ index@~.svelte
+│ │ │ ├ __layout@foo.svelte
+│ │ │ └ page.svelte
 │ │ └ __layout.svelte
-│ └ __layout.svelte
-└ __layout.svelte
+│ ├ __layout.svelte
+│ └ __layout-foo.svelte
 ```
 
-To reset _all_ layouts, including the home layout, a page can specify no layout, like this `index@.svelte` component:
+Layouts will inherit from 'sibling' layouts as well as parents — you'll never need to do this, but you _could_:
 
 ```
 src/routes/
-├ a/
-│ ├ b/
-│ │ ├ c/
-│ │ │ └ index@.svelte
-│ │ └ __layout.svelte
-│ └ __layout.svelte
-└ __layout.svelte
+├ __layout.svelte
+├ __layout-a@.svelte
+├ __layout-b@a.svelte
+├ __layout-c@b.svelte
+└ nested@c.svelte
 ```
 
-#### Inheritance chains
+Here, the inheritance chain would look like this (note that `__layout-a@` points at the unnamed default layout, because no name is specified):
 
-Layouts can themselves choose which other layouts they inherit from. Ordinarily, they will inherit from any default layouts 'above' them in the tree, but they can inherit named layouts using the same logic as pages. For example, `a/b/c/page.svelte` will inherit from `__layout-foo.svelte` and `a/b/c/__layout@foo.svelte` but _not_ `__layout.svelte` or `a/b/__layout.svelte`.
+```
+__layout.svelte
+  __layout-a@.svelte
+    __layout-b@a.svelte
+      __layout-c@b.svelte
+        nested@c.svelte
+```
+
+#### Resetting layouts
+
+Sometimes you need a page to use the 'home' layout, regardless of its URL. You can do this by creating a top-level named layout that inherits from the default layout by using `@` without a name:
 
 ```
 src/routes/
 ├ a/
 │ ├ b/
 │ │ ├ c/
-│ │ │ ├ __layout@foo.svelte
-│ │ │ └ page.svelte
+│ │ │ └ index@home.svelte
 │ │ └ __layout.svelte
-│ ├ __layout.svelte
-│ └ __layout-foo.svelte
+│ └ __layout.svelte
+│ __layout.svelte
+└ __layout-home@.svelte
```

Comment:
Could we just call it `__layout-home.svelte` instead?

Reply:
So, reading your recent comments it seems we are still unclear about the semantics of inheritance and the `@` symbol.

What's unanimous so far is that `__layout.svelte` is a default layout, `__layout-foo.svelte` is a layout named `foo`, and `__layout[-foo]@bar.svelte` is a layout that extends the nearest layout named `bar`. 

The confusing parts are:

What do we mean by `__layout-foo@.svelte`? There are two options:

1) A layout `foo` that inherits from the nearest unnamed layout, aka. the default layout;
2) A layout `foo` that inherits from _no_ layout, aka. a layout reset.

Likewise, what do we mean by `__layout-foo.svelte`?

1) A layout that extends the nearest `foo`.
2) A layout that extends the nearest `foo` or, if that doesn't exist, the nearest default layout.

We must pick the same answer for both questions, as they are opposite sides of the same issue. As it stands, the first option is implemented.

We could switch to the second option, but I'm not sure it'd be more intuitive. For instance, with option 1, `__layout-foo.svelte` is equivalent to `__layout-bar@foo.svelte`. With option 2, you'd have too look up a parent layout `foo` differently depending on usage: `@foo` means `foo`, while `__layout-foo` means `foo` _or_ default layout.

I'd prefer option 1 and just work on rephrasing the docs. 

That said... I know @Rich-Harris dislikes reserved names, and we tried not to go there, but at this point I'm wondering if we should just _not_ support `@.svelte` at all and make `default` a reserved name for the default layout. It'd be clearer than picking a meaning for the empty shorthand.

Reply:
> What do we mean by `__layout-foo@.svelte`?

If I read the docs correctly, then it's currently implemented as option 1. Layout resets are to be done using named layouts. I mostly agree with this, but question why `@` is needed at all in this case instead of just calling it `__layout-foo.svelte`

> Likewise, what do we mean by __layout-foo.svelte?

I'd go with option 1 here. And I'd probably have it throw an exception if `foo` does not exist.

Reply:
> If I read the docs correctly, then it's currently implemented as option 1.

Yes, that's correct.

> I mostly agree with this, but question why `@` is needed at all in this case instead of just calling it `__layout-foo.svelte`

Because if we keep option 1, `__layout-foo.svelte` would be a reset instead of inheriting from the default layout, and in this case the intention would be to have `__layout-home@.svelte` inherit from `__layout.svelte` so we could reset to "the root layout" instead of an empty one.

Reply:
I still believe the first option is easier to understand and, if we did consider reserving the `default` layout name, it'd be even easier. The whole ruleset would be:

- `__layout[-layout][@parent].svelte` defines a layout named `name = layout ?? 'default'` that extends the nearest layout named `parent ?? name`.
- `some-page[@parent].svelte` defines a page that extends the nearest layout named `parent ?? 'default'`.
- If `@parent` is explicitly specified it must be known, ie. `...@unknown.svelte` is an error.
- "Nearest layout named foo" means the first layout named `'foo'` found in the current folder or its parents.

In this case, `__layout-home@.svelte` wouldn't be a thing and we'd have to be explicit with `__layout-home@default.svelte`.

Reply:
Ok. I think that makes sense and if we just clarify that in the docs it should probably be sufficient

Reply:
I like being explicit with `default`

Reply:
closing as API was updated